contents/security/passwordStorage.md

```diff
@@ -0,0 +1,118 @@
+<frontmatter>
+  title: Introduction to Password Storage
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Introduction to Password Storage
+
+Author: [Jeremy Choo](https://github.com/ChooJeremy) <br>
+Reviewers: [Amrut Prabhu](https://github.com/amrut-prabhu), [Marvin Chin](https://github.com/marvinchin), [Tan Zhen Yong](https://github.com/Xenonym), [Wang Junming](https://github.com/junming403)
+
+## Overview
+
+While developing your own application, it is very common to implement login systems to be able to identify users. Whenever there is a need to store user-specific information, developers typically implement user accounts to identify these users separately. You might have seen this mechanism implemented on other websites - using registration forms and requring you to login with a username and password to identify you for the <trigger for="pop:session"><a href="https://support.google.com/analytics/answer/2731565?hl=en">session</a></trigger>.
+
+<popover id="pop:session" title="A _session_  is used to broadly describe a user's visit" placement="top">
+  <div slot="content">
+Generally, a session starts when a user visits your page, and ends when the user closes the browser window.
```

Comment:
Generally speaking it should be the browser window. If, for example, you login into IVLE (without checking the remember me box) and then close the tab and reopen a new tab for IVLE, it will keep you logged in.